index.js

```diff
@@ -21,6 +23,9 @@ const OPERATIONS = {
 }
 
 const defaultOptions = {
+  tempRetryTimes: 5,
+  tempRetryInterval: 50,
+  retryTemporaryErrors: true,
```

Comment:
I would argue this should be false by default, and clients should opt-in. We don't want to modify the default couchbase behaviour so drastically.